hawkrest/__init__.py

```diff
@@ -23,10 +23,44 @@
 default_message_expiration = 60
 
 
+def lookup_credentials(cr_id):
```

Comment:
Done, but I should point out that it was `lookup_credentials()` originally -- it's only showing as an addition because I moved it within the file. I've had to fix the reference to it in `test_middleware.py` as well.